cmd/kube-apiserver/app/server.go

```diff
@@ -127,6 +127,9 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	// Use protobufs for self-communication.
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
Hmm, unless not all apiservers will have protobuf support. @deads2k @sttts, thoughts?